src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -2633,6 +2641,10 @@ private void visitFunctionParameters(
       if (emitTemplatizedTypes) {
         visitTemplateTypes(ftype, alreadyEmittedTemplateType, true);
       }
+      // In partial mode when all the parameters to the function are unknown, it might be the
+      // case that the function is overriding/implementing a superclass that is in an unseen input.
```

Comment:
superclass or interface (for the next soul that thinks they can be smart about this).